components/brave_rewards/resources/internals/components/general.tsx

```diff
@@ -37,11 +37,13 @@ export class General extends React.Component<Props, {}> {
           paymentId={this.props.data.info.walletPaymentId}
           logEntries={this.props.data.eventLogs}
         />
-        <BalanceWrapper>
-          <Balance
-            balance={this.props.data.balance}
-            externalWallet={this.props.data.externalWallet} />
-        </BalanceWrapper>
+        {Boolean(this.props.data.info.bootStamp) &&
```

Comment:
nit: typically we wouldn't convert using `Boolean(...)` here since the logical expression will do that.

Reply:
Yeah, that's right, but oddly, if I write only `this.props.data.info.bootStamp && ...`, I get a visual artifact when refreshing the page repeatedly:
![image](https://user-images.githubusercontent.com/5390451/223283064-37269ce8-e12c-43b4-be81-927d970a27a4.png)
which I don't get with `Boolean(this.props.data.info.bootStamp) && ...`. It does go away after fully loading the page, but still it looks weird. 😬

Reply:
Oh, right - it's a timestamp number with "null" zero semantics. You might want to consider something like:

```js
this.props.data.info.bootStamp > 0
```